### PR TITLE
New Magic: The Gathering Spice

### PR DESCRIPTION
**Pull request by alohaas**
@moollaza still working on this, but I wanted to get the PR up to claim the idea :)
https://duck.co/ia/view/magic_the_gathering

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@alohaas I played around with this and it seems like all the functionality is there?

I think the code just needs a little cleanup and this should be mergeable :+1:

File: share/spice/magic_the_gathering/magic_the_gathering.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.zci--magic_the_gathering {
```
Comment (moollaza) [inline share/spice/magic_the_gathering/magic_the_gathering.css]: Let's remove this

File: share/spice/magic_the_gathering/magic_the_gathering.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                elClass: {
+                    tileFoot: "tx-clr--grey-light"
+                }
+            }
+        });
+    };
+    function pushArrayData(arr) {
+        
+    }
+    //function getInfoBoxData(item) {
```
Comment (moollaza) [inline share/spice/magic_the_gathering/magic_the_gathering.js]: Let's remove these comments

File: lib/DDG/Spice/MagicTheGathering.pm [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+use strict;
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
```
Comment (moollaza) [inline lib/DDG/Spice/MagicTheGathering.pm]: We can remove these boilerplate comments

**Comment by moollaza** [comment]
@alohaas I just realized you had this on a local branch so I cleaned it up a bit and added passing tests so we can deploy to beta :+1:

**Comment by alohaas** [comment]
@moollaza wow thanks! I was wanting to continue some design tweaks, but it would probably be best to get it live rather than procrastinate like I have been XD

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=fire magic card)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/magic_the_gathering_mobile.png)

**Comment by alohaas** [comment]
@moollaza thanks again.

One of the things I wasn't sure of was which information would be most important to include in the preview of the card versus the expanded view. There is also the issue of accuracy given that there are multiple editions of each card and I'm only selecting the first one. 

I was thinking of recruiting some developers in Slack who are more familiar with the game who can help me with investigate these UX questions. What do you think?

**Comment by moollaza** [comment]
> I was thinking of recruiting some developers in Slack who are more familiar with the game who can help me with investigate these UX questions. What do you think?

I think it's probably good enough to launch. If there are any major improvements to be made, the community can voice their opinions and fix things :+1:

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**